include/aws/s3/private/s3_client_impl.h

```diff
@@ -183,6 +191,14 @@ struct aws_s3_client {
     aws_s3_client_shutdown_complete_callback_fn *shutdown_callback;
     void *shutdown_callback_user_data;
 
+    /*
+     * Optional.
+     * Configuration for using proxy from environment variable.
```

Comment:
`from environment variable.`?

Is this a copy/paste error between `proxy_env_var_settings`?

Reply:
Yes, fixed. Thanks.